scipy/stats/tests/test_binned_statistic.py

```diff
@@ -105,6 +106,20 @@ def test_1d_range_keyword(self):
         assert_array_almost_equal(mean, mean_range2)
         assert_array_almost_equal(bins, bins_range2)
 
+    def test_1d_multi_values(self):
+        x = self.x
+        v = self.v
+        w = self.w
+
+        stat1v, edges1v, bc1v = binned_statistic(x, v, 'mean', bins=10)
+        stat1w, edges1w, bc1w = binned_statistic(x, w, 'mean', bins=10)
+        stat2, edges2, bc2 = binned_statistic(x, [v, w], 'mean', bins=10)
+
+        assert_array_almost_equal(stat2[0], stat1v)
+        assert_array_almost_equal(stat2[1], stat1w)
+        assert_array_almost_equal(edges1v, edges2)
+        assert_array_almost_equal(bc1v, bc2)
```

Comment:
Yeah, [looks like](http://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_almost_equal.html) you're right!